README.rst

```diff
@@ -27,6 +28,9 @@ For more information, read the `full documentation`_.
 If you just want a quick intro, watch this screencast demo of using CumulusCI to configure a Salesforce project from a GitHub repository:
 https://asciinema.org/a/91555
 
+Supported Python Versions: 3.6, 3.7, 3.8
```

Comment:
@davisagli - Wasn't sure where to exactly put this. Let me know if there's a standard place to include this, or if you think it would look better under a subheading.

Reply:
This is fine.